wiki/Registration/en.md

```diff
@@ -1,43 +1,39 @@
 How do I register?
 ------------------
 
-<center>
-**<font size="3">Please note that having more than one account at any time is an infringement of <span class="plainlinks"> \[</wiki/Game_Modes/osu!>!:Rules osu! rules\]</span>. Follow the below steps to set up your osu! account.**
-</font>
-<span style="font-size:18px">1. On the <span class="plainlinks"> [main webpage](https://osu.ppy.sh)</span>, click "<b>I'm new!</b>" to start the registration process. </span>
 
-<span style="font-size:18px">2. Fill out all fields in the form below and press "Continue" </span>
+**Please note that having more than one account at any time is an infringement of [Rules osu! rules](/wiki/Rules/). Follow the below steps to set up your osu! account.**
```

Comment:
`of Rules osu! rules`... wth?

---

Replace line with: `**Please note that having more than one account at any time is an infringement of the [osu! rules](/wiki/Rules/). Follow the below steps to set up your osu! account.**`